contracts/src/L1/rollup/ScrollChain.sol

```diff
@@ -165,13 +190,13 @@ contract ScrollChain is OwnableUpgradeable, PausableUpgradeable, IScrollChain {
         uint8 _version,
         bytes calldata _parentBatchHeader,
         bytes[] memory _chunks,
-        bytes calldata _skippedL1MessageBitmap
+        bytes calldata _skippedL1MessageBitmap,
+        uint64 _prevLastAppliedL1Block
```

Comment:
What is the first value of `_prevLastAppliedL1Block`? And except the first one, all `_prevLastAppliedL1Block` can be found in the `_parentBatchHeader`, right?